wremnants/recoil_tools.py

```diff
@@ -5,29 +5,31 @@
 import logging
 import sys
 import decimal
+import pathlib
 
 def drange(x, y, jump):
     while x < y:
         yield float(x)
         x += decimal.Decimal(jump)
         
-
+data_dir = f"{pathlib.Path(__file__).parent}/data/"
+        
 class Recoil:
 
     def __init__(self, type_):
     
         if type_ == "highPU":
         
             self.recoil_qTbins = list(drange(0, 30, 0.5)) + list(range(30, 60, 2)) + list(range(60, 100, 5)) + list(range(100, 210, 10)) + [10000]
-            rInput_binned = "wremnants/data/recoil_fits_Z.root"
+            rInput_binned = data_dir + "recoil_fits_Z.root"

```

Comment:
A similar implementation is in #67, sorry for the duplication

Reply:
okay then I wil drop it from this one.